infra/lib/api-service.ts

```diff
@@ -46,9 +46,13 @@ export function createAPIService(
   // Create a new Amazon Elastic Container Service (ECS) cluster
   const cluster = new ecs.Cluster(stack, "APICluster", { vpc, containerInsights: true });
 
-  // Create a Docker image and upload it to the Amazon Elastic Container Registry (ECR)
-  const dockerImage = new ecr_assets.DockerImageAsset(stack, "APIImage", {
-    directory: "../api/app",
+  // Create an ECR repo where we'll deploy our Docker images to, and where ECS will pull from
+  const ecrRepo = new Repository(stack, "APIRepo", {
+    repositoryName: "metriport/api",
+  });
+  new CfnOutput(stack, "APIECRRepoURI", {
+    description: "API ECR repository URI",
+    value: ecrRepo.repositoryUri,
   });
```

Comment:
maybe I'm missing something, but how will this work now?

the instructions say to deploy the infra, and then deploy the image, however isn't this a circular dependency? ie if the infra needs the image, but the image needs the infra.

Reply:
Yes, it is. The first deploy is going to be messy, then once the ECR is deployed, all will work. Now that I think about it, we could have this on a `BaseStack` and, like secrets, have it deployed before the APIStack; in between those, we would deploy the API's image to ECR, so when the APIStack deploys the ECR is populated.

Reply:
I suggest we leave as is, since we’d need to do the same for the Widget S3 bucket and that’d be harder b/c would require recreating the bucket.